syntax = "proto3";
package requestMsg;
message RequestMsg
{
    //1 密钥协商 //2 密钥校验; // 3 密钥注销
    int32 cmdType = 1; // 报文类型
    string clientId = 2; // 客户端编号
    string serverId = 3; // 服务器端编号
    string sign = 4;
    string data = 5;
}